### DISCUSSION TITLE
Shadowsocks for iPhone 15

### DISCUSSION THREAD

**Discussion by Piranaaa7** [shadowsocks/shadowsocks-org#260] [Q&A]
Hi everybody.
How can I run this on my iPhone.
I have shadow socks running on windows and Mac. Please assist thanks!!!

**Comment by irmo-de** [comment]
[You can use outline vpn.](https://apps.apple.com/us/app/outline-app/id1356177741)

**Reply by Activeleds** [reply]
You can set up bypass as well for the apps with the shadowsock .
See the edit then setting while on the check on app bypass 
Then set toggle to the apps you desire to use bypass and routes settings as well.
Hope this helps

**Comment by dmolnar110** [comment]
Go here. Translate in Google: https://github.com/relei1988/shadowsocks-2/blob/master/4-ios-setup-guide-en.md
Use [Potatso](https://apps.apple.com/us/app/potatso/id1239860606 ) its free and it works
